syntax = "proto3";
package organization;





/*
ListOrg返回
*/
message ListOrgResponse {
    /*
    org
    */
    int32 id = 1 ;
    /*
    过期时间戳
    */
    int32 expires = 2 ;
    /*
    创建日期
    */
    string createAt = 3 ;
    /*
    是否可用
    */
    bool valid = 4 ;
    /*
    更新事件戳
    */
    int32 ts = 5 ;
    /*
    名称
    */
    string name = 6 ;
    /*
    备注
    */
    string memo = 7 ;
}

/*
ListOrgApi返回
*/
message ListOrgResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    repeated organization.ListOrgResponse data = 4 ;
}

